### ISSUE TITLE
ProjectSelectionDialog already disposed error

### ISSUE THREAD

**Issue by etanshaul** [GoogleCloudPlatform/cloud-code-intellij#2647]
```
com.intellij.util.IncorrectOperationException at ObjectTree.register : Sorry but parent: com.google.cloud.tools.intellij.project.ProjectSelectionDialog$ProjectSelectionDialogWrapper@1054732b has already been disposed (see the cause for stacktrace) so the child: com.intellij.openapi.ui.ComponentValidator$$Lambda$3186/0x0000000101bad040@6f394d23 will never be disposed
Stack trace:
com.intellij.util.IncorrectOperationException: Sorry but parent: com.google.cloud.tools.intellij.project.ProjectSelectionDialog$ProjectSelectionDialogWrapper@1054732b has already been disposed (see the cause for stacktrace) so the child: com.intellij.openapi.ui.ComponentValidator$$Lambda$3186/0x0000000101bad040@6f394d23 will never be disposed
	at com.intellij.openapi.util.objectTree.ObjectTree.register(ObjectTree.java:61)
	at com.intellij.openapi.util.Disposer.register(Disposer.java:92)
	at com.intellij.openapi.ui.ComponentValidator.installOn(ComponentValidator.java:149)
	at com.intellij.openapi.ui.DialogWrapper.lambda$null$24(DialogWrapper.java:1973)
	at java.base/java.util.Optional.orElseGet(Optional.java:369)
	at com.intellij.openapi.ui.DialogWrapper.lambda$setErrorInfoAll$26(DialogWrapper.java:1973)
	at java.base/java.util.Collections$SingletonList.forEach(Collections.java:4854)
	at com.intellij.openapi.ui.DialogWrapper.setErrorInfoAll(DialogWrapper.java:1970)
	at com.intellij.openapi.ui.DialogWrapper.setErrorText(DialogWrapper.java:1935)
	at com.google.cloud.tools.intellij.project.ProjectSelectionDialog$ProjectSelectionDialogWrapper.setErrorText(ProjectSelectionDialog.java:471)
	at com.google.cloud.tools.intellij.project.ProjectSelectionDialog$1.lambda$onFailure$1(ProjectSelectionDialog.java:213)
	at java.desktop/java.awt.event.InvocationEvent.dispatch(InvocationEvent.java:313)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:776)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:727)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:721)
	at java.base/java.security.AccessController.doPrivileged(Native Method)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:85)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:746)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.java:817)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.java:766)
	at com.intellij.ide.IdeEventQueue.lambda$dispatchEvent$8(IdeEventQueue.java:405)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressMa
```

GoLand 2019.2.1
Plugin 19.7.2

**Comment by etanshaul** [comment]
@ivanporty I've seen this one a couple times

**Comment by axmerc** [comment]
Fixed by #1084
